README.md

```diff
@@ -6,111 +6,260 @@
 [![codecov](https://codecov.io/gh/xKDR/Survey.jl/branch/main/graph/badge.svg?token=4PFSF47BT2)](https://codecov.io/gh/xKDR/Survey.jl)
 [![Milestones](https://img.shields.io/badge/-milestones-brightgreen)](https://github.com/xKDR/Survey.jl/milestones)
 
+This package is used to study complex survey data. It aims to be a fast alternative
+to the [Survey package in R](https://cran.r-project.org/web/packages/survey/index.html)
+developed by [Professor Thomas Lumley](https://www.stat.auckland.ac.nz/people/tlum005).
 
-This package is used to study complex survey data. It aims to be a fast alternative to the [Survey package in R](https://cran.r-project.org/web/packages/survey/index.html) developed by [Professor Thomas Lumley](https://www.stat.auckland.ac.nz/people/tlum005).
+This package currently supports simple random sample, stratified sample, one- and
```

Comment:
It supports multistage sampling, but using single stage approximation.

Reply:
Right. I will add a short sentence mentioning this with a reference to the documentation for more details. Right now we don't have more details in the documentation, but we will add it. So the reference will lead nowhere useful for now, but will definitely lead somewhere soon.